role-make-version-changelog.sh

```diff
@@ -130,50 +135,86 @@ else
     fi
 fi
 if [ "$skip" = false ]; then
-    echo "If you want to continue, enter the new tag in the form ${allow_v}X.Y.Z"
-    echo "where X, Y, and Z are integers corresponding to the semantic"
-    echo "version based on the changes above."
-    echo "Or, just press Enter to skip this role and go to the next role."
+    ver_major=$(echo "${latest_tag}" | cut -d'.' -f 1)
+    ver_minor=$(echo "${latest_tag}" | cut -d'.' -f 2)
+    ver_patch=$(echo "${latest_tag}" | cut -d'.' -f 3)
+    commits=$(git log --pretty=format:%s --no-merges "${latest_tag}"..)
+    if echo "$commits" | grep -q '^.*\!:.*'; then
+        ver_major=$((ver_major+=1))
+        ver_minor=0
+        ver_patch=0
+    elif echo "$commits" | grep -q '^feat.*'; then
+        ver_minor=$((ver_minor+=1))
+        ver_patch=0
+    else
+        ver_patch=$((ver_patch+=1))
+    fi
+    new_tag=${allow_v}$ver_major.$ver_minor.$ver_patch
     while true; do
-        read -r -p "Old tag is ${latest_tag:-EMPTY} - new tag? (or Enter to skip) " new_tag
-        if [ -z "$new_tag" ]; then
+        read -r -p "The script calculates the new tag based on the above
+conventional commits.
+The previous tag is $latest_tag.
+The new tag is $new_tag.
+You have three options:
+1. To continue with the sugested new tag $new_tag, enter 'y',
```

Comment:
Previously, you could either enter a new tag to proceed, or press Enter to skip. I am adding an option to enter `y` to proceed with the suggested tag.

Reply:
I added some formatting, if it works correctly there should not be any input required from users to format changelog. I suggest using it as is for now - the script still opens changelog in editor for review. After we verify there are now bugs, we can skip this verification step.